README.md

```diff
@@ -55,6 +55,11 @@ directory ignored, as you don't want to commit generated files. In this case, th
 we don't want `ember` to be a dependency of installing `rustdoc`, and so we do commit
 those generated files.
 
+## Known issues (with solutions):
+
+ * "Which URL to call?": http://127.0.0.1:8000/index.html#/example
```

Comment:
This isn't really an issue, so much as something that should be moved into the actual docs of the README.